lib/linkify.dart

```diff
@@ -78,7 +81,8 @@ class LinkifyOptions {
 
 const _urlLinkifier = UrlLinkifier();
 const _emailLinkifier = EmailLinkifier();
-const defaultLinkifiers = [_urlLinkifier, _emailLinkifier];
+const _userTagLinkifier = UserTagLinkifier();
+const defaultLinkifiers = [_urlLinkifier, _emailLinkifier, _userTagLinkifier];
```

Comment:
```suggestion
const defaultLinkifiers = [_urlLinkifier, _emailLinkifier];
```

Since there's no direct linker for this, let's keep it out of the defaults